dotCMS/src/main/java/com/dotcms/security/apps/AppsKeyResetEventListener.java

```diff
@@ -0,0 +1,43 @@
+package com.dotcms.security.apps;
+
+import com.dotcms.system.event.local.model.EventSubscriber;
+import com.dotcms.system.event.local.type.security.CompanyKeyResetEvent;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.util.Logger;
+import java.io.IOException;
+
+/**
+ * Once registered this listener takes care of resetting the
+ */
+public class AppsKeyResetEventListener implements EventSubscriber<CompanyKeyResetEvent> {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A class which only has private constructors should be final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5677177)